tomato-timer/src/googleAuth.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { GoogleLogin } from 'react-google-login-component';
+
+class Login extends React.Component{
+  constructor (props, context) {
+    super(props, context);
+  }
+ 
+  responseGoogle (googleUser) {
+    
+    var id_token = googleUser.getAuthResponse().id_token;
+    console.log({accessToken: id_token});
```

Comment:
Careful logging this sensitive info to the console. I'd pull this out before you commit.